internal/pkg/scaffold/cmd.go

```diff
@@ -18,19 +18,28 @@ import (
 	"path/filepath"
 
 	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/input"
+	"github.com/operator-framework/operator-sdk/pkg/metrics"
 )
 
-const CmdFile = "main.go"
+const (
```

Comment:
Any reason for this change?

Reply:
Ah, I was initially including the constants in this file, hence the change, will revert this part.